libs/utils/android/benchmark.py

```diff
@@ -171,6 +175,8 @@ def __init__(self):
         self.log.info('=== TestEnv setup...')
         self.te = TestEnv(self._getBmConf())
         self.target = self.te.target
+        # TODO: Check if TestEnv already provides support for rebooting
+        self._reboot()
```

Comment:
In my opinion this function would be better to be taken outside of the benchmark run itself and maybe be used as a system function at the beginning of the test? It does work alright used here but for complex tests (6 governors x 6 tests) you'll have a reboot for each combination.